docs/installation/frameworks/react.md

```diff
@@ -500,7 +500,73 @@ Finally, exclude CKEditor 5 SVG and CSS files from `file-loader`. Find the last
 }
 ```
 
-#### Using CKEditor 5 source
+Now, your setup with `create-react-app` is complete. You can also check how to configure Vite in the next section or move straight to [Using the editor from source](#using-the-editor-from-source).
+
+### Vite
+
+This guide assumes that you use `create-vite` as your boilerplate. To get started with Vite and React, run the command below.
+
+```bash
+# npm 6.x
+npm create vite@latest ckeditor5-react-example --template react
+
+# npm7+, extra double-dash is needed:
+npm create vite@latest ckeditor5-react-example -- --template react
+```
+
+This command will install and execute `create-vite`, the official project scaffolding tool for Vite.
+
+#### Installing necessary packages
+
+Besides the CKEditor base and plugins, you need to install additional packages to use it from source with React and Vite: the Vite plugin, the official React component, and the default theme.
+
+<info-box>
+	Using the Vite plugin to build CKEditor 5 from the source in Vite is still in the experimental phase. We encourage you to test it and give us feedback. To read more about integration with Vite or its limitations, check the {@link installation/advanced/integrating-from-source-vite Integrating from source with Vite} guide.
+</info-box>
+
+Install necessary packages alongside the default theme using the following command.
+
+```
+npm install --save \ 
+	@ckeditor/vite-plugin-ckeditor5 \
+	@ckeditor/ckeditor5-react \
+	@ckeditor/ckeditor5-dev-utils \
```

Comment:
`dev-utils` is not needed when using Vite, as it is only used to import some webpack helpers for webpack config.